content/guides/accessibility/color-considerations.mdx

```diff
@@ -0,0 +1,144 @@
+
+- Pass color contrast requirements, and
+- Have the mechanism for accessing and changing the theme be fully WCAG compliant.
+
+This is the strongest guarantee we have that: 
+
+1. Content can be perceived—and therefore understood and used—across a wide range of vision conditions, and that
+2. The mechanism to adjust GitHub's theme can be operated by the widest range of input modes.
+
+This allows users to use GitHub in a logged-out state. If a user is logged in, it also enables them to address their own specific usability and access needs, as well as aesthetic desires.
+
+### Non-default themes
+
+[All sub-themes **do not** need to meet WCAG color contrast requirements](https://github.com/github/accessibility/blob/main/docs/coaching-recommendations/primer/color-modes.md#guidance), provided the default Day theme and its content is accessible. This guidance is derived from [WCAG 2.1 Technique G174](https://www.w3.org/WAI/WCAG21/Techniques/general/G174).
+
+Certain themes are specifically designed to create a low contrast effect, notably the Dark Dimmed theme. 
+
+In this use case, a user selecting this theme is likely intentionally expressing a desire for an experience with less visual contrast. Like other theme selections, this may be to satisfy aesthetic sensibilities, or access needs that are derived from visual conditions that make a higher contrast experience unusable. In the case of access needs, a higher contrast experience may actually be less accessible for the user.
+
+The availability of the Dark Dimmed theme treatment allows users to customize their GitHub experience without having to resort to browser extensions or other user-hosted modifications. Of note is the portability of this theme, in that it will automatically be available across multiple sessions and devices.
+
+While other sub-themes are not required to meet WCAG color contrast requirements, GitHub should strive to ensure they can be whenever possible. 
+
+As other sub-themes are not specifically designed to create a lessened contrast experience, there is less certainty that a user who is using the sub-theme has no contrast-related access needs. 
+
+Because of this, sub-themes such as Light/Dark Protanopia & Deuteranopia should strive to be WCAG color contrast compliant. This is especially relevant for Light/Dark High Contrast sub-themes.
+
+## Inputs
+
+Being able to perceive the boundaries of an input is vital. Meeting WCAG color contrast requirements helps to ensure the widest possible range of users can perform the data entry required to operate GitHub successfully.
+
+Broadly-speaking, there are four overall considerations for input contrast:
+
+1. Text,
+2. Borders, 
+3. Selected state contrast, and
```

Comment:
Do we want to add error state here?

Reply:
Yes! Added language on validation state here https://github.com/primer/design/pull/560/commits/36d01b6f463d290b81e4ea7614b2dab7d2346ee2